bom/pom.xml

```diff
@@ -346,6 +347,12 @@
         <version>${netty.tcnative.version}</version>
         <scope>runtime</scope>
       </dependency>
+      <!-- Jaeger support -->
+      <dependency>
+        <groupId>io.jaegertracing</groupId>
+        <artifactId>jaeger-client</artifactId>
```

Comment:
I think that we will need a works-with CQ for this, won't we?

Reply:
I guess this should be sufficient yes. I wouldn't really enable anything be default, as without a Jaeger server, it isn't really useful anyway.